gtsfm/loader/hilti_loader.py

```diff
@@ -98,13 +98,16 @@ def __init__(
 
         # Read the constraints from the lidar/constraints file
         all_constraints: Dict[Tuple[int, int], Constraint] = self.__load_constraints()
+        logger.info("[hilti_loader] Number of filtered constraints: %d", len(all_constraints))
         filtered_constraints: Dict[Tuple[int, int], Constraint] = self._filter_outlier_constraints(all_constraints)
```

Comment:
We should remove line 102 now? As we are already doing the filtering?

Reply:
As a sanity check, we can log # all_constraints, # filtered_constraints, # self._constraints. I have unit tested these, nothing should be removed, only updated.

Reply:
Will do